lib/unit/models/check_deposit/image_request.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Request to upload and get a check deposit image
+# @see https://docs.unit.co/check-deposits#upload-front-side-image
+# @see https://docs.unit.co/check-deposits#upload-back-side-image
+# @see https://docs.unit.co/check-deposits#get-specific-check-deposit-front-image
+# @see https://docs.unit.co/check-deposits#get-specific-check-deposit-back-image
+module Unit
+  module CheckDeposit
+    class ImageRequest
+      attr_reader :deposit_id, :file, :is_front_side
+
+      # @param deposit_id [String]
+      # @param file [String] - optional
```

Comment:
this is the file content or file name? lets be more explicit here, we saw it can be confusing in the applications